### ISSUE TITLE
Superfluous newlines in table column descriptions

### ISSUE THREAD

**Issue by chris-pieri** [typedoc2md/typedoc-plugin-markdown#591]
Hello 👋,

I'm using version: `4.0.0-next.55`

I am able to render a table for my interface with the following headers **Property**, **Type**, **Description**.

Currently any property with a **Default Value** gets added on a new line in the description cell. Is it possible to instead add a **Default Value** header in between **Type** and **Description**?

**Comment by tgreyuk** [comment]
Defaults values now exposed in table. Both of the following will work in `4.0.0-next.56`:

```
prop = 'propValue'
```

```
/**
 * @defaultValue 'propValue'
 * /
prop: string
```

**Comment by chris-pieri** [comment]
Thanks for making the update it's working now!

One nit picky thing, it looks like the `<br />` tags are still being added in the description cell for the rows that have a default value (tested on version `4.0.0-next.60`). 

Sorry, I tried uploading images but my work laptop won't allow it 🤦‍♂️.

**Comment by tgreyuk** [comment]
@chris-pieri  Thanks.  Just to confirm you mean the extra space below the description?

<img width="610" alt="Screenshot 2024-05-03 at 17 25 20" src="https://github.com/typedoc2md/typedoc-plugin-markdown/assets/11680870/3c558261-c651-4dc2-96ad-e8428b0f65df">

**Comment by chris-pieri** [comment]
Yes exactly, thanks

**Comment by tgreyuk** [comment]
fixed in typedoc-plugin-markdown@4.0.1

**Comment by chris-pieri** [comment]
Thank you looks great ! Congrats on 4.0 🎉
